cmd/hyperconverged-cluster-operator/main.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
```

Comment:
Please keep ony (up to) 4 import groups:
1. standard lib
2. general
3. kubevirt reos
4. hco repo.

Reply:
Done